### ISSUE TITLE
How to remove ESS style completely?

### ISSUE THREAD

**Issue by TheDohn** [emacs-ess/ESS#1286]
Spacemacs user here, so I won't even try bother you all with my config. In short, there seems to be some sort of interaction between `evil-paste` (from evil mode https://github.com/emacs-evil/evil) and `ess-r-mode` such that whenever I paste text, it gets formatted by ess. I am not clear if this is a feature or a bug, but it isn't really the behavior l desire. I have tried setting the `ess-style-alist` to a variety of values and none of them (including the custom option OWN) are what I want.

In short: Is there a way to completely disable the styling/linting of ESS mode?

**Comment by lionel-** [comment]
I'm an evil user as well and I'm not sure what you mean. The style you're referring to is only used when Emacs indents, but I don't think pasting is supposed to invoke indentation?

**Comment by TheDohn** [comment]
Thanks for the response! It is a bit hard to reproduce sometimes. But you can see in this video, when I yank some text, move down, and paste it, the `ggtitle(...)` shifts to the left, which is very strange. I assume something about spacemacs is activating a hook somewhere, but it is hard to track down...


https://github.com/emacs-ess/ESS/assets/43042967/6bf2fc4e-7524-4150-801b-557a9f50a1d8

**Comment by TheDohn** [comment]
This problem is showing up in other modes for me, so it isn't an ESS issue. Sorry for the confusion, but it is very very hard to reproduce, and at first seemed to impact ESS.
